src/main/kotlin/no/nav/familie/tilbake/service/KravgrunnlagTjeneste.kt

```diff
@@ -0,0 +1,34 @@
+package no.nav.familie.tilbake.service
+
+import no.nav.familie.tilbake.common.Periode
+import no.nav.familie.tilbake.kravgrunnlag.KravgrunnlagRepository
+import no.nav.familie.tilbake.kravgrunnlag.domain.Klassetype
+import no.nav.familie.tilbake.kravgrunnlag.domain.Kravgrunnlag431
+import no.nav.familie.tilbake.service.beregning.LogiskPeriodeTjeneste
+import no.nav.familie.tilbake.service.modell.LogiskPeriode
+import org.springframework.stereotype.Service
+import java.math.BigDecimal
+import java.util.SortedMap
+import java.util.UUID
+
+@Service
+class KravgrunnlagTjeneste(private val kravgrunnlagRepository: KravgrunnlagRepository) {
```

Comment:
Er ikkje dette noko Saikat måtte implementere for hentFakta-PR-en? Har han renama den?